packages/blocks/src/blocks.ts

```diff
@@ -1,29 +1,40 @@
 import {
```

Comment:
this file is pretty much the same, with some reorganizing of blocks, comments/links to docs, and moving of elements to own file.